<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_patient_DB</class>
 <widget class="QDialog" name="Create_patient_DB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание записи в базе данных</string>
  </property>
  <widget class="QLineEdit" name="APM_house_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_89">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>10. Адрес постоянного места жительства: область </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_90">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Профессия</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_to_BD_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_91">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>4. СНИЛС</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Document_for_l_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_92">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>360</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>кв.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_93">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>7. Отчество</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_94">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;2. Номер страхового полиса ОМС&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_95">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>540</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Должность</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_96">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>район</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_97">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>6. Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_98">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>населенный пункт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_99">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>улица</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_100">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>11. Адрес регистрации по месту пребывания: область</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_101">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>300</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>кв.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Izh_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_102">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>8. Пол </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_103">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>540</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Иждивенец</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Profession_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>540</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_104">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>корпус</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="APM_region_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="R_appartment_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_105">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>дом</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="R_housing_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>360</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_106">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>улица</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="R_area_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Job_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_107">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3. Код льготы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="APM_locality_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="APM_area_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="R_locality_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="APM_housing_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="R_house_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>360</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_108">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>14. Инвалидность</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_109">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5. Фамилия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_110">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>9. Дата рождения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_111">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>населенный пункт</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="R_street_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Post_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>540</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="APM_appartment_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_112">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>район</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_113">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1. Страховая медицинская организация</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="R_region_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDP_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="CB_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_114">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>15. Место работы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SNILS_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="Female_radio_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_115">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>12. Телефон: домашний</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MN_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Official_phone_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_116">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rod</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>МЕДИЦИНСКАЯ КАРТА АМБУЛАТОРНОГО БОЛЬНОГО №</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_117">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>13. Документ, удостоверяющий право на льготное обеспечение (наименование, №, дата, кем выдан)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="disability_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>480</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Policy_number_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_118">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>дом</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Surname_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Home_phone_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="Male_radio_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>М</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="APM_street_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SMO_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_119">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>служебный</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Birthday_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_120">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>корпус</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
